app_dart/lib/src/request_handlers/push_engine_status_to_github.dart

```diff
@@ -112,6 +115,11 @@ class PushEngineStatusToGithub extends ApiRequestHandler<Body> {
         case Task.statusFailed:
         case Task.statusInfraFailure:
           log.debug('Using ${task.status} from commit ${task.commitSha} ref ${task.ref} builder ${task.builderName}');
+          await luciBuildService.checkRerunBuilder(
```

Comment:
Let's log that we are retrying the task.

Reply:
Added to `luci_build_service` when rerun is actually needed.